packages/mdc-button/mdc-button.scss

```diff
@@ -41,6 +41,16 @@
   }
 }
 
+.mdc-button--raised,
+.mdc-button--unelevated,
+.mdc-button--raised,
```

Comment:
should --raised be listed twice?

Reply:
Whoops... clearly I misread a bit when I was looking at the selectors below to apply this to.